roles/graphql-api/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+- name: configure nginx upstream
+  become: true
+  template: src=upstream.conf.j2 dest=/etc/nginx/conf.d/upstream/http_fxa_graphql_api.conf
+  notify: reload nginx config
+
+- name: pull fxa-graphql-api docker image
+  become: true
+  docker_image:
+    # pull image always. In a pending update to ansible, it will only set
+    # changed:true if the image has actually changed. However, docker_container
+    # `state:started` will only re-start if image of configuration has
+    # changed.
+    force: true
+    state: present
+    name: mozilla/fxa-graphql-api
+    tag: "{{ graphql_api_docker_tag }}"
+  register: image
+
+- debug: var=image
+
+- name: Start fxa-graphql-api docker container
+  become: true
+  docker_container:
+    # See note in https://github.com/mozilla/fxa-dev/tree/docker#about-using-docker_container-and-quoting-of-environment-values
+    name: graphql-api
+    image: mozilla/fxa-graphql-api{{ ':' + graphql_api_docker_tag }}
+    state: "{{ graphql_api_docker_state }}"
+    network_mode: host
+    ports:
+      - "{{ graphql_api_private_port }}:{{ graphql_api_private_port }}"
+    command: node dist/bin/main.js
+    env:
+      # using development to start; may switch to production
+      NODE_ENV: "development"
+      AUTH_SERVER_URL: "{{ auth_public_url }}"
+      PROFILE_SERVER_URL: "{{ profile_public_url }}"
+      ACCESS_TOKEN_REDIS_HOST: 127.0.0.1
+      SENTRY_DSN: ''
+      AUTH_MYSQL_DATABASE: fxa
+      AUTH_MYSQL_HOST: 127.0.0.1
+      AUTH_MYSQL_USERNAME: root
+      AUTH_MYSQL_PASSWORD: ''
+      OAUTH_MYSQL_DATABASE: fxa_oauth
```

Comment:
Oops. Forgot to use vars for these config items (so it works when fxa-dev uses a remote rds database). I'll work on that change now.

Reply:
updated to use variables